data/levels/world2/worldmap.stwm

```diff
@@ -16,6 +16,7 @@
       (z-pos 100)
       (x 1120)
       (y 1376)
+      (sprite "images/worldmap/halloween/tree1.png")
```

Comment:
This is redundant. Can you please delete it

Reply:
sure